test/libsolidity/syntaxTests/controlFlow/uninitializedAccess/bug10821.sol

```diff
@@ -0,0 +1,13 @@
+contract Test {
+
+  struct Sample { bool flag; }
+  Sample public s;
+
+   function testFunc() external {
+        if(true){}
+        Sample storage t;
+        t.flag=true;
+    }
+}
+// ----
+// TypeError 3464: (155-156): This variable is of storage pointer type and can be accessed without prior assignment, which would lead to undefined behaviour.
```

Comment:
Could you please add the test case in the original bug report as well?

This one

```
contract Test {
  
  struct Sample { bool flag; }
  Sample public s;
  
  modifier checkAddr(address _a){
      require(_a!=address(0));
      _;
  }
  function testFunc(address _a) external checkAddr(_a) {
        Sample storage t;
        t.flag=true;
    }  
}
```